draft-ietf-dnsop-svcb-https.md

```diff
@@ -1332,7 +1336,7 @@ The domain owner could add this record:
     @ 7200 IN HTTPS 1 . alpn=h3
 
 to indicate that https://simple.example supports QUIC
-in addition to TLS over TCP (the implicit default).
+in addition to HTTP/1.1 over TLS over TCP (the implicit default).
```

Comment:
```suggestion
in addition to HTTP over TLS over TCP (the implicit default,
with HTTP/1.1 or HTTP/2 negotiated during the TLS handshake 
via ALPN).
```

It could be either as ALPN will negotiate.

Reply:
Yeah, but the sentence is: "The domain owner could add this record ... to indicate that it ... supports ...".  The record indicates support for HTTP/3 and HTTP/1.1, not HTTP/2.  An HTTP/2-only would not attempt to use this record, because there is no indication that the endpoint supports HTTP/2.